### PR TITLE
Palindrome-doubly linked list in python

### PR DESCRIPTION
**Pull request by Swetha126**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes i have read the guidelines carefully

### Description

I have added a new file called Palindrome-doublylinkedlist under ds folder in python.
This is the code with appropriate comments and format for checking if the given string is a palindrome or not using linked lists in python.
I have updated the readme file as well. 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

I would like to contibute this code to issue #2420

### CODE REVIEW

File: Python/ds/Palindrome-doublylinkedlist.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# Sample Input/Output :-
+# Test case 1:
+# Enter the string : clouds
+# Not a Palindrome
+# Test case 2:
+# Enter the string : madam
+# Is a Palindrome
+# Test case 3:
+# Enter the string : books
+# Not a Palindrome
```
Comment (HarshCasper) [inline Python/ds/Palindrome-doublylinkedlist.py]: Use a Multi-Line String

File: Python/ds/Palindrome-doublylinkedlist.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+        self.prev = None
+
+
+class DoublyLinkedList:
+
+    @staticmethod
+    def printList(node):
```
Comment (HarshCasper) [inline Python/ds/Palindrome-doublylinkedlist.py:45]: Add a function docstring

File: Python/ds/Palindrome-doublylinkedlist.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        rev_key = ""
+        while node:
+            rev_key += node.data
+            node = node.next
+        return rev_key
+
+    def __init__(self):
+        self.head = None
+
+    def push(self, new_data):
```
Comment (HarshCasper) [inline Python/ds/Palindrome-doublylinkedlist.py:57]: Add a function docstring

**Review by HarshCasper — CHANGES_REQUESTED** [review]
One more change: Push your Commits into one single commit.

File: Python/ds/Palindrome-doublylinkedlist.py (hunk 1 above)
Comment (Swetha126) [reply Python/ds/Palindrome-doublylinkedlist.py]: I have made the  change by adding the sample test cases on top of the file

File: Python/ds/Palindrome-doublylinkedlist.py (hunk 2 above)
Comment (Swetha126) [reply Python/ds/Palindrome-doublylinkedlist.py:45]: I have added the necessary docstring

File: Python/ds/Palindrome-doublylinkedlist.py (hunk 3 above)
Comment (Swetha126) [reply Python/ds/Palindrome-doublylinkedlist.py:57]: I have added the necessary docstring

**Comment by Swetha126** [comment]
I am unable to merge all my commits into a single commit as i do not have full access to this repository and i worked on this particular changes from git browser.

image

Please use this option to merge all my commits into a single commit while merging the pull request.
Thank you!

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**